deploy/sample-backup-configs/sample_config_hwdb.ini

```diff
@@ -8,38 +8,48 @@
 # Only AGFIs for the latest release of FireSim are guaranteed to be available.
```

Comment:
@jerryz123 i didn't manually check if this is the case or not, but just FYI, you can do 
in firesim/deploy/
cat built-hwdb-entries/* >> FILE.ini

it would be good to commit these in that order if so, it makes it easier to diff updates in the future.

Reply:
You mean order alphabetically?

Reply:
yeah

Reply:
These are currently not the right AGFIs. 
I'll update AGFIs once all other changes are approved.

Reply:
@jerryz123 this is now resolved right?

Reply:
Yes